packages/rainbowkit/src/wallets/Wallet.ts

```diff
@@ -33,6 +33,7 @@ type RainbowKitConnector<C extends Connector = Connector> = {
 };
 
 export type Wallet<C extends Connector = Connector> = {
+  accentColor?: string;
```

Comment:
I wonder if this should be called `iconAccent` so that it forms a pair with `iconBackground`? This might also avoid potential confusion that it's used as a general accent color in the UI.

Reply:
gotcha, will rename for clarity